addons/link_tracker/controller/main.py

```diff
@@ -9,11 +9,14 @@ class LinkTracker(http.Controller):
 
     @http.route('/r/<string:code>', type='http', auth='public', website=True)
     def full_url_redirect(self, code, **post):
-        country_code = request.session.geoip and request.session.geoip.get('country_code') or False
```

Comment:
You can't do that, http_routing is not a dependency of link_tracker, it will crash

Reply:
Hi rde, thanks for your fast review! I guess I should have marked it as draft, as I was still testing if the bot filter worked properly. I have moved the is_a_bot function in tools :).

Reply:
Oh, ok :-)
I figured since there were reviews going on and it was open, it was approaching shipping time :D 
Saw the mail notification about this pr since it changes http routing so I opened it out of curiosity :)